include/vicon_estimator.h

```diff
@@ -159,7 +177,7 @@ class RotationalEstimatorParameters {
   Eigen::Vector3d initial_dorientation_estimate_;
   Eigen::Vector3d initial_drate_estimate_;
   double outlier_threshold_degrees_;
-  int maximum_outlier_count_;
+  int maximum_outlier_count_rotation_;
```

Comment:
Because the maximum outlier count is a member of the class "RotationalEstimatorParameters" naming this variable "maximum_outlier_count_rotation_" is a bit redundant. I would leave it as "maximum_outlier_count".